src/cryptonote_basic/cryptonote_format_utils.cpp

```diff
@@ -146,6 +146,8 @@ namespace cryptonote
     if (tx.version >= 2 && !is_coinbase(tx))
     {
       rct::rctSig &rv = tx.rct_signatures;
+      if (rv.type == rct::RCTTypeNull)
```

Comment:
Is this related?

Reply:
This is what made me find that bug in the first place. I guess it can be omitted since all null rct types are coinbase so will be caught by the previous check. But pedantically it's best to check the rct data exists before we get it below.